### PR TITLE
Document and provide examples of potential approaches for providing inline live code in design system.

### PR DESCRIPTION
**Pull request by sheaphillips**

### CODE REVIEW

File: app-web/src/components/ComponentPreview/ComponentPreview.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                return <div>Error {error.message}</div>
+            } else if (!isLoaded) {
+                return <div>Loading...</div>
+            } else {
+                return (
+                    <iframe src={'data:text/html;base64,' + sampleContent} frameBorder={'0'} />
+                );
+            }
+    }
+}
```
Comment (patricksimonian) [inline app-web/src/components/ComponentPreview/ComponentPreview.js]: add new line

File: app-web/src/hoc/withNode.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+            this.state = {
+                node: node
+            }
+        }
+
+        render() {
+            return <WrappedComponent {...this.props} node={this.state.node}/>;
+        }
+    }
+}
```
Comment (patricksimonian) [inline app-web/src/hoc/withNode.js]: new line at end

**Comment by patricksimonian** [comment]
There appears to be a ton of changes that are already merged into master in this PR, maybe a rebase will fix this?

**Comment by sheaphillips** [comment]
We've moved work on this to a separate PR. Closing this one.

**Pull request closed without merging**